src/eva/vision/data/wsi/patching/samplers/grid.py

```diff
@@ -0,0 +1,47 @@
+"""Grid sampler."""
+
+from typing import Generator, Tuple
+
+from eva.vision.data.wsi.patching.samplers import _utils, base
+
+
+class GridSampler(base.Sampler):
```

Comment:
do you think there will be need for this one?

Reply:
Maybe for datasets such as BACH this might come in useful (not for the slide level task, but the patch level dataset as the patches there are too big and downscaling them to 224 as we currently do leads to loss of information and mpp's the FMs are usually not trained on much).